### ISSUE TITLE
Training loss

### ISSUE THREAD

**Issue by YuhsiHu** [kwea123/CasMVSNet_pl#32]
Hi, thank you for your code! I have some questions:
1. Is there any change between your code and the original code of cascade-stereo?
2. what is the final training loss of Cascade MVSNet on DTU dataset?

**Comment by kwea123** [comment]
I believe there is some small hyperparam difference, but the model structure is the same. I didn't read their code so I cannot tell you where is different exactly.
You can download the pretrained log and see the loss.
